main_app/templates/base.html

```diff
@@ -1,61 +1,73 @@
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
+  </head>
+  <body class="d-flex flex-column min-vh-100">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="{% url 'home' %}">
+          <i class="fa-solid fa-synagogue fa-lg"></i> &nbsp;&nbsp;StockSense
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarColor01" aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarColor01">
+          <ul class="navbar-nav me-auto">
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Features</a>
+                <div class="dropdown-menu">
+                  <a class="dropdown-item" href="{% url 'about' %}">About</a>
+                  <a class="dropdown-item" href="#">Contact</a>
+                  <a class="dropdown-item" href="#">Pricing</a>
+                </div>
+              </li>
+              {% if user.is_authenticated %}
+            <li class="nav-item">
+              <a class="nav-link" href="{% url 'home' %}">
+                <i class="fa-solid fa-house fa-lg"></i> Home
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="">
+                <i class="fa-solid fa-circle-plus fa-lg"></i> Add Item
+              </a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="">
+                  <i class="fa-solid fa-file-circle-plus fa-lg"></i> Add category
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="">
+                  <i class="fa-solid fa-sitemap fa-lg"></i> All categories
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="{% url 'logout' %}">
+                  <i class="fa-solid fa-arrow-right-from-bracket fa-lg"></i> Logout
+                </a>
+              </li>
+              {% endif %}
+          </ul>
+        </div>
       </div>
     </nav>
-  </header>
-  <main class="container">
-    {% block content %}
-    
-    {% endblock %}
-  </main>
-  <footer class="page-footer">
-    <div class="right">All Rights Reserved, &copy; 2024 Inventory Management System &nbsp;</div>
-  </footer>
-</body>
+    <div class="container-fluid">
+      {% block content %}
+      {% endblock %}
+    </div>
+    <footer class="mt-auto mb-4">
+      <div class="text-center">
+        <span>
+          Copyright &copy; <script>document.write(new Date().getFullYear())</script>&nbsp;&nbsp;StockSense: Inventory Management System
```

Comment:
This footer will keep changing to the current year, right? Shouldn't it just be the year we developed the app?